public/src/overmind/actions.tsx

```diff
@@ -490,52 +506,76 @@ export const rebuildSubmission = async ({ state, actions, effects }: Context, {
 
 /* rebuildAllSubmissions rebuilds all submissions for a given assignment */
 export const rebuildAllSubmissions = async ({ effects }: Context, { courseID, assignmentID }: { courseID: bigint, assignmentID: bigint }): Promise<boolean> => {
-    const response = await effects.grpcMan.rebuildSubmissions(assignmentID, courseID)
+    const response = await effects.api.client.rebuildSubmissions({
+        courseID,
+        assignmentID,
+    })
     return success(response)
```

Comment:
Looks like we might end up removing the `success` function; in which case this should return `!response.error`